### DISCUSSION TITLE
Audio: Failed to install Steam audio drivers

### DISCUSSION THREAD

**Discussion by mikicof** [LizardByte/community#363] [Sunshine]
Hi,

I just migrated from Gamestream and did a clean install. In the configuration I’ve changed only the Install Steam Audio Drivers to Enabled as the desc says that it will enable surround sound and muting, but none of them works.
Checking the logs I see a warning on each connection saying:
`Warning; Failed to install Steam audio drivers: 259`

I have steam installed and tried installing Steam Link audio drivers but with no change. I’ve seen older answers using VBCable and setting virtual sink but with the current documentation seems that none of this should be necessary.

Thank you!

**Comment by ReenigneArcher** [comment]
Is steam installed to the default location?

**Reply by mikicof** [reply]
Yes, both of them on C:/ drive also

**Comment by awvandyke** [comment]
I am having a similar issue, also have them installed in the default locations. In my case, I'm getting the error on my OLED deck but my gf's LCD gets audio as well as connecting via the iOS app.

**Reply by Trimod** [reply]
`Warning: Failed to install Steam audio drivers: 259`
Same issue, audio is played from both host and client. Streaming from Windows 11 PC to Nvidia Shield TV Pro, using Sunshine and Moonlight.

Steam link is able to mute host ok on same devices.

[a comment by an automated account is not shown]
